source/Popup/styles.scss

```diff
@@ -0,0 +1,51 @@
+@import "../styles/fonts";
+@import "../styles/reset";
+@import "../styles/variables";
+
+@import "~webext-base-css/webext-base.css";
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap");
```

Comment:
We cant rely on an internet connection - all files required by the client add-on must be local.

Reply:
okay sure